mocks/routes/applications.js

```diff
@@ -75,16 +75,20 @@ const updateApplication = {
       id: "success",
       response: {
         status: 204,
-        body: applications[0],
+        body: applications[4],
       },
     },
     {
       id: "simulate",
       response: (req, res) => {
         const applicationId = req.params.id;
-        const application = applications.find((applicationData) => applicationData.id === applicationId);
+        const application = applications.find((applicationData) => {
```

Comment:
Not sure why this part has changed?

Reply:
`application` would return undefined for some odd reason. The change fixed it for me.